src/styles/blocks/main.scss

```
@@ -0,0 +1,9 @@
+.main {
+  padding: 0 20px;
+  max-width: 1260px;
+  margin: 0 auto;
```

```suggestion
  margin: 0 auto;
```

You cannot use outer spacing (margins) for BEM blocks.
You may use mix of styles like
`<main class="main main__container">` and give margins to `__container` element:
```
.main {
  
 &__container {
   padding: 0 20px;
   max-width: 1260px;
   margin: 0 auto;
```